pallets/parachain-staking/src/set.rs

```diff
@@ -42,7 +43,7 @@ impl<T: Ord> OrderedSet<T> {
 	///
 	/// `v` will be sorted and dedup first.
 	pub fn from(mut v: Vec<T>) -> Self {
-		v.sort();
+		v.sort_by(|a, b| b.cmp(a));
```

Comment:
Would be nice to have a comment that explicitly states that reverse ordering is wanted here. Also the OrderedSet gets less generic by that.

Reply:
Agree on the comment. I also agree with the OrderedSet getting less generic in general but not for this specific line of code.

Reply:
Should be fixed in 211fabd.